README.md

```diff
@@ -1,9 +1,73 @@
 # aptpac
 
-A simple wrapper for `pacman` with a syntax similar to `apt` to help people transitioning to arch and arch based distros like manjaro.
+AptPac is a program which helps with the transition to Arch Linux and similar.
 
-## Installation
-Go to the <a href="https://github.com/Itai-Nelken/aptpac/releases/latest" target="_blank">latest release</a> and follow the instructions there.
+It simplifies using pacman as it works like the easier to use APT package manager found in Debian based distributions.
 
-## Uninstallation
-Go to the <a href="https://github.com/Itai-Nelken/aptpac/releases/latest" target="_blank">latest release</a> and follow the instructions there.
+The program comes in 2 variants, a bash shell version, and a C code variant.
+The C code variant should be used since C code is quicker and better to use rather than shell scripts.
+
+## Usage
+
+``` bash
+aptpac install $PACKAGENAME 
+aptpac search $PACKAGENAME
+aptpac remove $PACKAGENAME
+```
+
+## **Installation**
+
+Since there are 2 variants of aptpac, there are 2 install methods, please choose the variant you would like, C or Bash
```

Comment:
```suggestion
Since there are 2 variants of aptpac, there are 2 install methods, please choose the variant you would like: C (recommended) or Bash
```